extensions/wikia/Search/classes/Services/SearchCores.php

```diff
@@ -10,8 +10,9 @@ class SearchCores {
 	// list all common field ids that can be shared across cores
 	const F_WIKI_ID = 'wikiId';
 	const F_WIKI_HOST = 'wikiHost';
+
 	public static function getCoreFieldNames( $core ) {
-		$core_opt = [ ];
+		$core_opt = [];
 		switch ( $core ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this "switch" statement with "if" statements to increase readability. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1301)